Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -2242,48 +2307,24 @@ private static bool HandleKeywords(Material destMat, string keyword)
                 infoRecord.instanceObjects.Add(instancedCopy);
             }
         }
-        else {
-            // This instance exists inside another instanced object,
-            // update its transform to be correct relative to the parent on the infoRecord:
-            // For example, in an object structure like this:
-            // MainObject
-            //   InstancedChildA
-            //     InstancedChildB
-            // For the instance renderer method, InstancedChildA and InstancedChildB exist as
-            // MeshSyncInstanceRenderer components on MainObject and the transforms in data.transforms
-            // are relative to MainObject. 
-            // In this mode, where we use copies of the objects, InstancedChildB already exists as child of InstancedChildA
-            // but the transform of InstancedChildB needs to be updated to be relative to MainObject.
-
-            if (data.transforms.Length != 1) {
-                Debug.LogWarning(
-                    $"[MeshSync] There should never be more than 1 transform on instances inside other instances: {data.path}");
-                return;
-            }
-
-            // Find the instanced copy as (a potentially indirect) child of `MainObject`:
-            var instancedCopy =
-                FilmInternalUtilities.GameObjectUtility.FindByPath(instanceRendererParent.transform, data.path);
-
-            if (instancedCopy == null) {
-                Debug.LogWarning(
-                    $"[MeshSync] No instanced copy found for {data.path}");
-                return;
-            }
-
-            // Set the transform relative to `MainObject`.
-            // The easiest and safest way to do that is to:
-            // - parent the instanced copy to `MainObject`
-            // - set the transform matrix
-            // - restore the original parent
-            var originalParent = instancedCopy.parent;
+    }
 
-            instancedCopy.transform.SetParent(instanceRendererParent.transform);
+    static void SetInstanceTransformRelative(
```

Comment:
Where is this function called ?

Reply:
We're not using it anymore, I've removed it now.